src/avfoundation.cs

```diff
@@ -9672,6 +9681,20 @@ interface AVCaptureDevice {
 		[Export ("activeColorSpace", ArgumentSemantic.Assign)]
 		AVCaptureColorSpace ActiveColorSpace { get; set; }
 
+		// From AVCaptureDevice (AVCaptureDeviceDepthSupport) Category
+
+		[iOS (11,0)]
```

Comment:
there are some API decorated with `[Mac]` attributes so it looks like the type is available on macOS and that 11,0 is not enough (if it's available it's missing 10,13 or it's not available).

Reply:
The whole thing is inside a `#if !MONOMAC` block